core/src/main/java/org/geysermc/geyser/translator/level/block/entity/CampfireBlockEntityTranslator.java

```diff
@@ -40,10 +40,12 @@ public class CampfireBlockEntityTranslator extends BlockEntityTranslator {
     @Override
     public void translateTag(NbtMapBuilder builder, CompoundTag tag, int blockState) {
         ListTag items = tag.get("Items");
-        int i = 1;
-        for (Tag itemTag : items.getValue()) {
-            builder.put("Item" + i, getItem((CompoundTag) itemTag));
-            i++;
+        if (items != null) {
```

Comment:
Take this line and the line above for `if (tag.get("Items") instanceof ListTag items)`?